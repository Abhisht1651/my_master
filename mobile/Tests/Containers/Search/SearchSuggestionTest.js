// @flow
import 'react-native'
import React from 'react'
import { shallow } from 'enzyme'
import TestFile from '../../../App/Containers/Search/SearchSuggestion'

describe('Search Suggestion Screen', () => {
  it('should render the component itself correctly', () => {
    const wrapper = shallow(<TestFile />)
    expect(wrapper).toMatchSnapshot()
  })
  it('should render the Text Component', () => {
    const wrapper = shallow(<TestFile />)
    expect(wrapper.find('Text')).toHaveLength(4)
  })
  it('should render the Image Component', () => {
    const wrapper = shallow(<TestFile />)
    expect(wrapper.find('Image')).toHaveLength(3)
  })
})
