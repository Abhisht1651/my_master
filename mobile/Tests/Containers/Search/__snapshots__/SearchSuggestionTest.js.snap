// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Suggestion Screen should render the component itself correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "color": "#1F1F60",
        "fontFamily": "BananaGrotesk-Regular",
        "fontSize": 32,
        "letterSpacing": -0.11,
        "opacity": 1,
        "textAlign": "left",
      }
    }
  >
    Last Searches
  </Text>
  <View
    style={
      Object {
        "marginTop": 6.5,
      }
    }
  >
    <View
      key="0"
      style={
        Object {
          "alignItems": "center",
          "borderBottomColor": "#E6ECED",
          "borderBottomWidth": 1,
          "flexDirection": "row",
          "justifyContent": "space-between",
          "minHeight": 136,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#8F8FAF",
            "fontFamily": "BananaGrotesk-Light",
            "fontSize": 28,
            "letterSpacing": 0,
            "opacity": 1,
            "textAlign": "left",
          }
        }
      >
        Amazonas fire
      </Text>
      <TouchableOpacity
        activeOpacity={0.2}
        delayLongPress={600}
        extraButtonProps={
          Object {
            "rippleColor": "transparent",
          }
        }
        onPress={[Function]}
      >
        <Image
          source={
            Object {
              "default": "",
            }
          }
          style={
            Object {
              "height": 22,
              "opacity": 1,
              "width": 22,
            }
          }
        />
      </TouchableOpacity>
    </View>
    <View
      key="1"
      style={
        Object {
          "alignItems": "center",
          "borderBottomColor": "#E6ECED",
          "borderBottomWidth": 1,
          "flexDirection": "row",
          "justifyContent": "space-between",
          "minHeight": 136,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#8F8FAF",
            "fontFamily": "BananaGrotesk-Light",
            "fontSize": 28,
            "letterSpacing": 0,
            "opacity": 1,
            "textAlign": "left",
          }
        }
      >
        Queen Elizabeth
      </Text>
      <TouchableOpacity
        activeOpacity={0.2}
        delayLongPress={600}
        extraButtonProps={
          Object {
            "rippleColor": "transparent",
          }
        }
        onPress={[Function]}
      >
        <Image
          source={
            Object {
              "default": "",
            }
          }
          style={
            Object {
              "height": 22,
              "opacity": 1,
              "width": 22,
            }
          }
        />
      </TouchableOpacity>
    </View>
    <View
      key="2"
      style={
        Object {
          "alignItems": "center",
          "borderBottomColor": "#E6ECED",
          "borderBottomWidth": 1,
          "flexDirection": "row",
          "justifyContent": "space-between",
          "minHeight": 136,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#8F8FAF",
            "fontFamily": "BananaGrotesk-Light",
            "fontSize": 28,
            "letterSpacing": 0,
            "opacity": 1,
            "textAlign": "left",
          }
        }
      >
        Nicolas Maduro
      </Text>
      <TouchableOpacity
        activeOpacity={0.2}
        delayLongPress={600}
        extraButtonProps={
          Object {
            "rippleColor": "transparent",
          }
        }
        onPress={[Function]}
      >
        <Image
          source={
            Object {
              "default": "",
            }
          }
          style={
            Object {
              "height": 22,
              "opacity": 1,
              "width": 22,
            }
          }
        />
      </TouchableOpacity>
    </View>
  </View>
</View>
`;
